src/map/quest.cpp

```diff
@@ -179,6 +179,7 @@ uint64 QuestDatabase::parseBodyNode(const YAML::Node &node) {
 				target->element = ELE_ALL;
 				target->mapid = -1;
 				target->map_name = "";
+				target->count_slave = true;
```

Comment:
I am not sure if we should make it true by default, like you also said in your comment on this PR.